src/to-do.test.js

```diff
@@ -0,0 +1,97 @@
+/** @jest-environment jsdom */
+import Display from './to-do.js';
+
+describe('Add new task', () => {
+  beforeEach(() => {
+    localStorage.clear();
+    jest.clearAllMocks();
+    localStorage.setItem.mockClear();
+    document.body.innerHTML = `
+    <div class="todo-body"></div>
+    <div class="todo-edit flex">
+              <input
+                type="text"
+                class="no-outline type-task"
+                placeholder="Add to your list..."
+                value="visit the gym" 
+              />
+              <i class="enter-btn fa-solid fa-right-from-bracket"></i>
+            </div>`;
+  });
+
+  test('Adding todo object to the local storage', () => {
+    const addTodoOnSpy = jest.spyOn(Display, 'addTodo');
+    Display.addTodo();
+    expect(addTodoOnSpy).toHaveBeenCalledTimes(1);
+    const result = JSON.parse(window.localStorage.getItem('todo'));
+    expect(result.length).toBe(1);
+    expect(result[0].text).toBe('visit the gym');
+  });
+
```

Comment:
- [ ] Good job so far :+1: Please make sure that all your tests are passing.
![Screenshot from 2023-01-26 21-58-33](https://user-images.githubusercontent.com/92303891/214961066-c12fd6a1-ad04-4b1a-9d5e-1a85ed196d4e.png)